ios/Capacitor/Capacitor/WebViewAssetHandler.swift

```diff
@@ -121,6 +135,63 @@ public class WebViewAssetHandler: NSObject, WKURLSchemeHandler {
         return false
     }
 
+    func handleCapacitorHttpRequest(_ urlSchemeTask: WKURLSchemeTask, _ localUrl: URL, _ isHttpsRequest: Bool) {
+        var urlRequest = urlSchemeTask.request
+        let url = urlRequest.url!
+        var targetUrl = url.absoluteString
+            .replacingOccurrences(of: CapacitorBridge.httpInterceptorStartIdentifier, with: "")
+            .replacingOccurrences(of: CapacitorBridge.httpsInterceptorStartIdentifier, with: "")
+
+        // Only replace first occurrence of the scheme
+        if let range = targetUrl.range(of: self.serverUrl?.scheme ?? InstanceDescriptorDefaults.scheme) {
+            targetUrl = targetUrl.replacingCharacters(in: range, with: isHttpsRequest ? "https" : "http")
+        }
+
+        urlRequest.url = URL(string: targetUrl)
+
+        let urlSession = URLSession.shared
+        let task = urlSession.dataTask(with: urlRequest) { (data, response, error) in
+            urlSession.invalidateAndCancel()
```

Comment:
Calling this does nothing: https://developer.apple.com/documentation/foundation/urlsession/1411538-invalidateandcancel

Reply:
Well, TIL. I don't know I ever had occasion to use `invalidateAndCancel` but thinking about it, it makes sense since you really have no idea what `shared` could be up to and the system would have to restart the shared session if you don't.